non-msrv-tests/tests/ui/no_generics.stderr

```diff
@@ -0,0 +1,15 @@
+error: derive-where doesn't support items without generics, `skip` attributes or `enum`s implementing `Default`, as this can already be handled by standard `#[derive(..)]`
+ --> tests/ui/no_generics.rs:2:1
+  |
+2 | / #[derive_where(Clone)]
+3 | | struct Test1(u8);
+  | |_________________^
+
+error: derive-where doesn't support items without generics, `skip` attributes or `enum`s implementing `Default`, as this can already be handled by standard `#[derive(..)]`
```

Comment:
We should keep in mind, that default might in the future be std: https://doc.rust-lang.org/beta/unstable-book/language-features/derive-default-enum.html

Reply:
I saw that, we could really use `#[cfg(version(..))]`.

Reply:
yeah, we can make this an issue for now to keep this in mind